app/src/App.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import Board from './Board'
+import './App.css'
+
+class App extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      game: new Board(25, 25),
+      playInterval: null,
+    }
+    this.click = this.click.bind(this)
+    this.step = this.step.bind(this)
+    this.play = this.play.bind(this)
+    this.randomize = this.randomize.bind(this)
+    this.clear = this.clear.bind(this)
+  }
+
+  click(event) {
+    const aliveState = event.target.className
+    const coord = event.target.getAttribute('dataset').split(',')
```

Comment:
I believe it's simpler to keep row and column in separate attributes: `event.target.dataset.row` and `event.target.dataset.col` - that way you don't need to ever split/join them (cleaner).